### PR TITLE
New Medikit sprites and a Randomized medkit

### PR DESCRIPTION
**Pull request by PestoVerde322**
## About The Pull Request

This PR will implement brand new Sprites for the medikits, replacing the now outdated 45° icons for them, adding alongside new variations.
In this Pr there's also a Randomized medikit containing the content of ALL the medikits (except the syndicate one).
this also include a new sprite for the medical aid kit, now properly telegraphed to be a high capacity medikit instead of a normal one, this comes with an alternative version on a dedicated item which is a Doctor's bag (see the image).

Alternative medikit Icons for the brute, burn, air, toxins and Advanced medikit done by AdipemDragon over in /tg/ (based off from https://github.com/tgstation/tgstation/pull/65803)

## Why It's Good For The Game

Old sprites bad, new sprites good, Randomized medikit better.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

BRAND NEW

</details>

## Changelog
:cl:
add: Added a random medkit for Battle royale and Hope reasons.
add: Added a doctor's bag as a dedicated alternative item for the medical aid kit in the future.
imageadd: Brand new sprites for all the medkits and their In-hand icons.
/:cl:

### CODE REVIEW

**Comment by Fronsis** [comment]
Love them, good job as usual Pigeon!!

**Comment by PestoVerde322** [comment]
NOTE UP AHEAD: I've updated the Icons for the medikits to be more sharp, on top of that I've added brand new icons for the in-hand version.
Meem2

File: code/game/objects/items/storage/firstaid.dm [hunk 1]
```diff
@@ -263,14 +296,69 @@
...
+		/obj/item/storage/pill_bottle/kelotane)
+	var/items_inside = list(
+		pick(supplies) = 1,
+		pick(supplies) = 1,
+		pick(supplies) = 1,
+		pick(supplies) = 1,
+		pick(supplies) = 1,
+		pick(supplies) = 1,
+		/obj/item/healthanalyzer = 1)
+	generate_items_inside(items_inside,src)
```
Comment (PowerfulBacon) [inline code/game/objects/items/storage/firstaid.dm]: At this point use
```
new /obj/item/healthanalyzer(src)
for(var/i in 1 to 6)
	var/selected_type = pick(supplies)
	new selected_type(src)
```

File: code/game/objects/items/storage/firstaid.dm [hunk 2]
```diff
@@ -280,14 +368,14 @@
...
-	new /obj/item/reagent_containers/pill/patch/silver_sulf(src)
-	new /obj/item/reagent_containers/pill/patch/silver_sulf(src)
-	new /obj/item/clothing/glasses/hud/health/night(src)
+	var/static/items_inside = list(
+		/obj/item/stack/medical/gauze = 1,
+		/obj/item/defibrillator/compact/combat/loaded = 1,
+		/obj/item/reagent_containers/hypospray/combat = 1,
+		/obj/item/reagent_containers/pill/patch/styptic = 2,
+		/obj/item/reagent_containers/pill/patch/silver_sulf = 2,
+		/obj/item/clothing/glasses/hud/health/night = 1)
```
Comment (PowerfulBacon) [inline code/game/objects/items/storage/firstaid.dm]: Does this overfill the medkit, since a gauze is being added

Comment (PestoVerde322) [reply code/game/objects/items/storage/firstaid.dm]: oops, wrong math, yeah i can remove the gauze since it's quite a common find anyway

**Review by PowerfulBacon — DISMISSED** [review]

**Comment by Sinmero** [comment]
Im not a big fan of the new icons on the medkit. The cross is fine because it is easily recognisable and universal. 
The toxin kit used to have a biohazard sign (which is recognizable and gives you an idea of what is inside) now its some blue thing.
The radiation sign could be crispier compaired to the old one.
The rest signs telegraph random things. Like blue lock looking thing on the brutekit or exclamation mark.

If you look at this from a perspective of a new player it is likely medkits with this signs would be confused with other objects.

**Comment by BrumalLake** [comment]
> new player ... confused with other objects.

read item name

File: code/game/objects/items/storage/firstaid.dm [hunk 3]
```diff
@@ -263,14 +296,64 @@
 		/obj/item/storage/pill_bottle/penacid = 1)
 	generate_items_inside(items_inside,src)
 
+/obj/item/storage/firstaid/advanced/Initialize(mapload)
+	. = ..()
+	icon_state = pick("firstaid-advanced","firstaid-advancedalt")
+
+//First Random kit
+/obj/item/storage/firstaid/random
+	name = "mistery medical kit"
```
Comment (PowerfulBacon) [inline code/game/objects/items/storage/firstaid.dm]:
```suggestion
	name = "mystery medical kit"
```

**Review by PowerfulBacon — APPROVED** [review]

**Pull request merged**